cmd/updater/subcommands/status/status.tmpl

```diff
@@ -0,0 +1,8 @@
+Datadog Updater v{{ .Version }}
+{{ range $name, $package := .Packages }}
+{{ $name }}
+  State: {{ yellowText "unknown (unimplemented)" }}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏Maybe add a todo and/or a follow-up ticket?